stock_batch_picking/wizard/batch_picking_creator.py

```diff
@@ -43,14 +43,9 @@ def _default_picker_id(self):
         else:
             warehouse = self.env['stock.warehouse'].search([
                 ('company_id', '=', self.env.user.company_id.id)
-            ], limit=1)
+            ], limit=1)[:1]
```

Comment:
Aren't we achieving the desired result already with the `limit=1`? The result of this should be either a singleton or an empty recordset which would eval to False.